### PR TITLE
removed duplicate params from salesforce integration

### PR DESCRIPTION
**Pull request by tomneeman151293**
## Contributing to Cortex XSOAR Content
Make sure to register your contribution by filling the [contribution registration form](https://forms.gle/XDfxU4E61ZwEESSMA)

**The Pull Request will be reviewed only after the contribution registration form is filled.**

## Status
- [x] In Progress
- [ ] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: link to the issue

## Description
When fixing issue 
https://github.com/demisto/etc/issues/34397#event-4455028219
and making sure integrations are validated without duplicate params
this integration was found invalid.
Removed the duplicated params.

## Does it break backward compatibility?
   - [X] Yes
       - Further details: Not sure if it breaks, but I am a bit concerned how removing params might affect users.

## Must have
- [ ] Tests
- [ ] Documentation

### CODE REVIEW

**Comment by ShahafBenYakir** [comment]
Do we have an issue on this integration? or you have found this using your fix to validate?

**Review by ShahafBenYakir — APPROVED** [review]

File: Packs/SalesforceIndicators/ReleaseNotes/1_0_1.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+
+#### Integrations
+##### Salesforce Indicators
+- Removed duplicated params.
```
Comment (amshamah419) [inline Packs/SalesforceIndicators/ReleaseNotes/1_0_1.md:4]:
```suggestion
- Fixed an issue where the *Indicator Reputation* parameter was duplicated in the integration configuration.
```

Comment (amshamah419) [reply Packs/SalesforceIndicators/ReleaseNotes/1_0_1.md:4]: @tomneeman151293 just commit these changes and you're good to go

Comment (ShahafBenYakir) [reply Packs/SalesforceIndicators/ReleaseNotes/1_0_1.md:4]: And the `Fetch indicators` also

Comment (amshamah419) [reply Packs/SalesforceIndicators/ReleaseNotes/1_0_1.md:4]: Fixed

**Pull request merged**